webvh/webvh/resolver/resolver.py

```diff
@@ -0,0 +1,132 @@
+"""DID Resolver for Cheqd."""
+
+import json
+import logging
+from dataclasses import dataclass
+from typing import Optional, Pattern, Sequence, Text
+
+from acapy_agent.config.injection_context import InjectionContext
+from acapy_agent.core.profile import Profile
+from acapy_agent.resolver.base import (
+    BaseDIDResolver,
+    DIDNotFound,
+    ResolverError,
+    ResolverType,
+)
+from aiohttp import ClientSession
+from pydid import DIDDocument
+
+from ..validation import WebVHDID
+
+LOGGER = logging.getLogger(__name__)
+
+
+@dataclass
+class DIDLinkedResourceWithMetadata:
+    """Schema for DID Linked Resource with metadata."""
+
+    resource: dict
+    metadata: dict
+
+
+class DIDWebVHResolver(BaseDIDResolver):
```

Comment:
This class is injected and overrides the resolver in acapy with the addition of resolving/dereferencing the resources?